### ISSUE TITLE
Add a gg funnel function

### ISSUE THREAD

**Issue by Lingtax** [Lingtax/ggmeta#1]
This has some existing implementations (e.g. [ce-funnel](https://github.com/jsakaluk/ce-funnel)).

John also has a blog post on this: https://sakaluk.wordpress.com/2016/02/16/7-make-it-pretty-plots-for-meta-analysis/ 

I suggest it should have options including:

- Contour-enhanced funnel, line at 0, shaded regions for (p<0.1 & p > 0.05) & (p < 0.05 & p > 0.01)
- Standard funnel, line on summary effect size 

Point shape by grouping variable may be useful.

**Comment by ekothe** [comment]
@Lingtax might be work looking at @jsakaluk's repo on contour enhancing funnel plots https://github.com/jsakaluk/ce-funnel

**Comment by Lingtax** [comment]
Yep, that's why there's a link to that at the very top of this issue @ekothe  😛

**Comment by Lingtax** [comment]
Wolfgang also has an implementation here: https://github.com/wviechtb/metaggplots/
